### ISSUE TITLE
excluding entities

### ISSUE THREAD

**Issue by brunnels** [hwestphal/spring-boot-odata#1]
Thank you for your example.  AFAIK you have the only example with spring boot that is not using jaxrs.  I have your solution working with the latest jhipster with only a little modification.

The problem I'm running into now I think is that I have a very complex set of entities and I don't really need them all exposed through odata.  Do you know of a way to exclude specific entities via annotation or through the jpa service factory?

**Comment by brunnels** [comment]
I found the root cause of the issue I'm having is a bug with entity embedded id  classes. 
 https://issues.apache.org/jira/browse/OLINGO-546

I think I figured out how to exclude entities via META-INF/persistence.xml

**Comment by brunnels** [comment]
I got this worked out and mostly working in my scenario.  I've found quite a few bugs that I've had to fix like incorrect total count when doing server side paging.  It's also not easy to add additional data types like LocalDataTime and ZonedDateTime without replacing library classes with my own implementations.

I would like to look at https://github.com/sdl/odata-jpa-datasource when I get some time to see if it's any better.

Thanks again for sharing your example.  It helped a great deal to get me started on the right track.
